src/unix/core.c

```diff
@@ -35,7 +35,7 @@
 #include <sys/un.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
-#include <limits.h> /* INT_MAX, PATH_MAX */
+#include <limits.h> /* INT_MAX, PATH_MAX, IOV_MAX */
```

Comment:
not the right place to be asking questions about `IOV_MAX`. 

that should be determine on a system-to-system basis. This header should not be consulted here.